app/services/data/get-staging-omic-workload.js

```diff
@@ -28,41 +28,77 @@ module.exports = function (range) {
 
+      // OMIC Teams Licence cases Suspended
+      'omic_teams.licencetierd0s', 'omic_teams.licencetierd1s', 'omic_teams.licencetierd2s', 'omic_teams.licencetierd3s',
+      'omic_teams.licencetierc0s', 'omic_teams.licencetierc1s', 'omic_teams.licencetierc2s', 'omic_teams.licencetierc3s',
+      'omic_teams.licencetierb0s', 'omic_teams.licencetierb1s', 'omic_teams.licencetierb2s', 'omic_teams.licencetierb3s',
+      'omic_teams.licencetiera0s', 'omic_teams.licencetiera1s', 'omic_teams.licencetiera2s', 'omic_teams.licencetiera3s',
+
       // OMIC Teams Custody cases
       'omic_teams.custtier0',
       'omic_teams.custtierd0', 'omic_teams.custtierd1', 'omic_teams.custtierd2', 'omic_teams.custtierd3',
       'omic_teams.custtierc0', 'omic_teams.custtierc1', 'omic_teams.custtierc2', 'omic_teams.custtierc3',
       'omic_teams.custtierb0', 'omic_teams.custtierb1', 'omic_teams.custtierb2', 'omic_teams.custtierb3',
       'omic_teams.custtiera0', 'omic_teams.custtiera1', 'omic_teams.custtiera2', 'omic_teams.custtiera3',
 
+      // OMIC Teams Custody cases Suspended
+      'omic_teams.custtierd0s', 'omic_teams.custtierd1s', 'omic_teams.custtierd2s', 'omic_teams.custtierd3s',
+      'omic_teams.custtierc0s', 'omic_teams.custtierc1s', 'omic_teams.custtierc2s', 'omic_teams.custtierc3s',
+      'omic_teams.custtierb0s', 'omic_teams.custtierb1s', 'omic_teams.custtierb2s', 'omic_teams.custtierb3s',
+      'omic_teams.custtiera0s', 'omic_teams.custtiera1s', 'omic_teams.custtiera2s', 'omic_teams.custtiera3s',
+
       // T2A Community cases
       't2a.commtier0 AS t2a_commtier0',
       't2a.commtierd0 AS t2a_commtierd0', 't2a.commtierd1 AS t2a_commtierd1', 't2a.commtierd2 AS t2a_commtierd2', 't2a.commtierd3 AS t2a_commtierd3',
       't2a.commtierc0 AS t2a_commtierc0', 't2a.commtierc1 AS t2a_commtierc1', 't2a.commtierc2 AS t2a_commtierc2', 't2a.commtierc3 AS t2a_commtierc3',
       't2a.commtierb0 AS t2a_commtierb0', 't2a.commtierb1 AS t2a_commtierb1', 't2a.commtierb2 AS t2a_commtierb2', 't2a.commtierb3 AS t2a_commtierb3',
       't2a.commtiera0 AS t2a_commtiera0', 't2a.commtiera1 AS t2a_commtiera1', 't2a.commtiera2 AS t2a_commtiera2', 't2a.commtiera3 AS t2a_commtiera3',
 
+      // T2A Community cases Suspended
+      't2a.commtierd0 AS t2a_commtierd0s', 't2a.commtierd1 AS t2a_commtierd1s', 't2a.commtierd2 AS t2a_commtierd2s', 't2a.commtierd3 AS t2a_commtierd3s',
+      't2a.commtierc0 AS t2a_commtierc0s', 't2a.commtierc1 AS t2a_commtierc1s', 't2a.commtierc2 AS t2a_commtierc2s', 't2a.commtierc3 AS t2a_commtierc3s',
+      't2a.commtierb0 AS t2a_commtierb0s', 't2a.commtierb1 AS t2a_commtierb1s', 't2a.commtierb2 AS t2a_commtierb2s', 't2a.commtierb3 AS t2a_commtierb3s',
+      't2a.commtiera0 AS t2a_commtiera0s', 't2a.commtiera1 AS t2a_commtiera1s', 't2a.commtiera2 AS t2a_commtiera2s', 't2a.commtiera3 AS t2a_commtiera3s',
+
       // T2A Licence cases
-      't2a.licencetier0 AS t2a_licencetier0',
+      't2a.licencetier0 AS t2a_licencetier0s',
```

Comment:
This looks wrong .
I don't think this line should have changed.

Reply:
fixed, thx